Libplanet.Action.Tests/Mocks/MockAccount.cs

```diff
@@ -0,0 +1,121 @@
+using System.Numerics;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Libplanet.Store.Trie;
+using Libplanet.Types.Assets;
+using Libplanet.Types.Tx;
+
+namespace Libplanet.Action.Tests.Mocks
+{
+    public static class MockAccount
+    {
+        public static IAccount Empty { get; } =
+            new Account(default, new MerkleTrie(new MemoryKeyValueStore()));
+
+        public static IAccount SetBalance(
+            this IAccount account,
+            Address address,
+            FungibleAssetValue amount) =>
+            SetBalance(account, (address, amount.Currency), amount);
+
+        public static IAccount SetBalance(
```

Comment:
I think return `type` should be the same as the class not `IAccount`.

Reply:
but this is some kind of extension method for `IAccount`. note `this IAccount account`.

Reply:
Yes, and I'm suggesting this shouldn't be an extension for an `IAccount` 🙃.
`IAccount`'s behavior is complicated enough as it is. It isn't at all clear how "setting a balance" is achieved.
As any kind of change to balance requires `IActionContext`, this hides implicit `IActionContext` creation. Also, if the currency in question has minters, I don't think there is any way to set balance to a non-minter `Address`.

Reply:
Okay, then use which implementation in `IUnRecordableTrie` in `MockAccount`? `new UnRecordableTrie(new MerkleTrie())`?